src/yukon/backend/src/devserv/mock_server.py

```diff
@@ -0,0 +1,423 @@
+
+                self.session_scheduler.enter(
+                    status_event['timestamp_end'], 1, self.sse_builder, (data, 'NODE_STATUS', ))
+
+    def schedule_publishers_events(self, node: int, node_event: dict) -> None:
+        """
+        Node publishers changes
+        Currently only simulates adding or removing. No rate assotiated events
+        """
+
+        if 'publishers' in node_event:
+            data = {
+                "id": int(node),
+                "publishers": []
+            }
+
+            for idx2, pub in enumerate(node_event['publishers']):
+                assert isinstance(data['publishers'], list)
+
+                data['publishers'].append(pub)
+                data['publishers'][idx2].update(
+                    {"active": 1})
+
+                # subject line appears (assuming there's a corresponding subscriber)
+                if 'timestamp_start' in pub and pub['timestamp_start']:
+                    self.session_scheduler.enter(
+                        pub['timestamp_start'], 1, self.sse_builder, (data, 'NODE_STATUS', ))
+
+                # subject line disappears
+                if 'timestamp_end' in pub and pub['timestamp_end']:
+                    data['publishers'][idx2].update(
+                        {"active": 0})
+
+                    self.session_scheduler.enter(
+                        pub['timestamp_end'], 1, self.sse_builder, (data, 'NODE_STATUS', ))
+
+    def schedule_subscribers_events(self, node: int, node_event: dict) -> None:
+        """
+        Node subscribers changes
+        Currently only simulates adding or removing. No rate assotiated events
+        """
+        if 'subscribers' in node_event:
+            data = {
+                "id": int(node),
+                "subscribers": []
+            }
+
+            for idx2, sub in enumerate(node_event['subscribers']):
+                assert isinstance(data['subscribers'], list)
+
+                data['subscribers'].append(sub)
+                data['subscribers'][idx2].update(
+                    {"active": 1})
+
+                # subject line appears (assuming there's a corresponding publisher)
+                if 'timestamp_start' in sub and sub['timestamp_start']:
+                    self.session_scheduler.enter(
+                        sub['timestamp_start'], 1, self.sse_builder, (data, 'NODE_STATUS', ))
+
+                # subject line disappears
+                if 'timestamp_end' in sub and sub['timestamp_end']:
+                    self.session_scheduler.enter(
+                        sub['timestamp_end'], 1, self.sse_builder, (data, 'NODE_STATUS', ))
+
+    def sse_builder(self, data: str, event_type: str) -> None:
+        """
+        Generates the SSE reponse in a JSON format.
+        """
+        self.event = ServerSentEvent(
+            data=data, event=event_type)
+
+
+if __name__ == "__main__":
+    sys.stdout.write('\n\033[92mMock Backend process started!\n\033[0m')
```

Comment:
Consider [colorama](https://pypi.org/project/colorama/) or (better) [coloredlogs](https://pypi.org/project/coloredlogs/) (the latter is a recommended optional dependency for PyUAVCAN also).

Reply:
Done. Though I am not sure why the quart loggers output with and without the colour scheme at the same time. So I kept it commented for further investigation.